### DISCUSSION TITLE
Problem creating a new page

### DISCUSSION THREAD

**Discussion by SDGArmenia** [open-sdg/open-sdg#2085] [Q&A] [answered]
Hello!

We have a problem with a creating new page.
We want to and new page Dashboard near the FAQ on home page.
https://sdg.armstat.am/

In site repository  -    sdg-site-armenia/tree/develop/_pages
We created dashboard.md files in 3 languages. 

In circleci we get a following problem 

- ./_test/sdg-site-armenia/dashboard/index.html
  *  internally linking to /sdg-site-armenia/am/dashboard/, which does not exist (line 240)
     <a class="dropdown-item" href="/sdg-site-armenia/am/dashboard/">
              ՀԱՅԵՐԵՆ
            </a>
  *  internally linking to /sdg-site-armenia/am/dashboard/, which does not exist (line 260)
     <a class="language-toggle-link" href="/sdg-site-armenia/am/dashboard/">
                ՀԱՅԵՐԵՆ
            </a>
  *  internally linking to /sdg-site-armenia/ru/dashboard/, which does not exist (line 244)
     <a class="dropdown-item" href="/sdg-site-armenia/ru/dashboard/">
              Pусский
            </a>
  *  internally linking to /sdg-site-armenia/ru/dashboard/, which does not exist (line 264)
     <a class="language-toggle-link" href="/sdg-site-armenia/ru/dashboard/">
                Pусский
            </a>

HTML-Proofer found 4 failures!

Exited with code exit status 1

We have
******
In      circleci/config.yml  file we have 

Python CircleCI 2.0 configuration file
version: 2
jobs:
  test:
    docker:
      - image: cimg/ruby:2.6

*******
source "https://rubygems.org"

gem "jekyll", "3.8.4"
gem "html-proofer", "3.19.4" 
gem "jekyll-remote-theme"
gem "deep_merge"
gem "jekyll-open-sdg-plugins", "2.2.0"

Please , help to solve the problem

**Comment by sarah-beasley** [comment]
Hi, I think this problem could be solved by adding the correct permalinks within each language's dashboard file, as they're not pointing directly to the folder as the [written guidance suggests](https://open-sdg.readthedocs.io/en/latest/tutorials/create-pages/). I think it should say: 


---
title: Панель оценки тенденций
permalink: ru/dashboard/
language: ru
layout: page
---


So it knows that it's in the ru folder.

You will need to do the same for the hy dashboard file too.

I think what may also help is in the english file, also including the following line in its equivalent section:
language: en

Once this has been successfully created, you can then add a menu item in your site_config.yml file by following the [written guidance too](https://open-sdg.readthedocs.io/en/latest/tutorials/create-pages/).

Hope this helps, but let us know if you're still having a problem!

**Comment by SDGArmenia** [comment]
Thanks for the help

**Reply by SDGArmenia** [reply]
Thank you very much for your help.
We successfully created pages in 3 languages. But now we have another problem with code, whish we want to write in dashboard.md file.
We need to put the following code in the file

https://github.com/armstat/sdg-site-armenia/edit/create-pages-tutorial/_pages/dashboard.md

<script type="module" src="https://public.tableau.com/javascripts/api/tableau.embedding.3.latest.min.js" ></script>
<div style="align-items: center; display: flex; justify-content: center;" >  <tableau-viz hide-tabs="" id="tableauViz" src="https://public.tableau.com/views/SDG-ARM-2024/Story1?" hide-tabs="true", toolbar="hidden", height="600px", width="800px" > </tableau-viz></div>

We did it, but in result we have got the problem of the <tableau-viz> tag.   
https://sdg.armstat.am/dashboard/

![image](https://github.com/open-sdg/open-sdg/assets/46884106/5d5e9494-e06e-4fee-9ba9-6bcabbf1aac1)


In **index.html** file the code get the follow view


<div style="align-items: center; display: flex; justify-content: center;">  &lt; tableau-viz hide-tabs="" id="tableauViz" src="https://public.tableau.com/views/SDG-ARM-2024/Story1?" hide-tabs="true", toolbar="hidden", height="600px", width="800px" &gt; &lt;/ tableau-viz&gt;</div> 

In code is absent the tag <>  for <tableau-viz>.

Could you please , help us to solve the problem.

Thank you in advance

**Comment by christoph-LSN** [comment, marked as answer]
@SDGArmenia It works with adding the following Code:

I added a page:

---
title: Tests
permalink: /decision_makers/
layout: page
---

```
<div class='tableauPlaceholder' id='viz1710329228303' style='position: relative'>
    <noscript>
        <a href='#'>
            <img alt='Story 1 ' src='https:&#47;&#47;public.tableau.com&#47;static&#47;images&#47;SD&#47;SDG-ARM-2024&#47;Story1&#47;1_rss.png' style='border: none' />
        </a>
    </noscript>
    <object class='tableauViz' style='display:none;'>
        <param name='host_url' value='https%3A%2F%2Fpublic.tableau.com%2F' />
        <param name='embed_code_version' value='3' />
        <param name='site_root' value='' />
        <param name='name' value='SDG-ARM-2024&#47;Story1' />
        <param name='tabs' value='no' />
        <param name='toolbar' value='yes' />
        <param name='static_image' value='https:&#47;&#47;public.tableau.com&#47;static&#47;images&#47;SD&#47;SDG-ARM-2024&#47;Story1&#47;1.png' />
        <param name='animate_transition' value='yes' />
        <param name='display_static_image' value='yes' />
        <param name='display_spinner' value='yes' />
        <param name='display_overlay' value='yes' />
        <param name='display_count' value='yes' />
        <param name='language' value='de-DE' />
    </object>
</div>
<script type='text/javascript'>
    var divElement = document.getElementById('viz1710329228303');
    var vizElement = divElement.getElementsByTagName('object')[0];
    vizElement.style.width='1116px';
    vizElement.style.height='1091px';
    var scriptElement = document.createElement('script');
    scriptElement.src = 'https://public.tableau.com/javascripts/api/viz_v1.js';
    vizElement.parentNode.insertBefore(scriptElement, vizElement);
</script>
```

![image](https://github.com/open-sdg/open-sdg/assets/94443931/d136d141-366a-44f6-a5f3-d84635795dc7)


Visit the results here: https://christoph-lsn.github.io/MT_Site/decision_makers/index.html

One Question: How do you get the data from SDG in Tableau? By Hand?

**Comment by SDGArmenia** [comment]
Thank you very much. We have used your code and now its work.

Regarding the question you raised on the data in Dashboard we are pleased to inform you that  they are the source of the SDG data in Armstat's site, which we have combined in an excel file.
